### PR TITLE
Bes l1 streams definitions

### PR DESCRIPTION
**Pull request by BenEsquis**
Adding definitions of the L1 streams

### CODE REVIEW

File: scripts/scdf-shell/l1b/destroy.shell [hunk 1]
```diff
@@ -0,0 +1 @@
+stream destroy --name ${S2USER}-l0-raw-to-l0c
```
Comment (glmxndr) [inline scripts/scdf-shell/l1b/destroy.shell:1]: --name should be "${S2USER}-l1b-refining"

File: scripts/scdf-shell/inventory-l1b/create.shell [hunk 2]
```diff
@@ -0,0 +1,6 @@
+stream create \
+  --name "${S2USER}-l1b-inventory-ds" \
+  --definition ":${S2USER}-l1b-refined-context > inventory-l1b-ds"
+stream create \
+  --name "${S2USER}-l0-inventory-gr" \
```
Comment (glmxndr) [inline scripts/scdf-shell/inventory-l1b/create.shell:5]: Stream name seems incoherent.

File: scripts/scdf-shell/inventory-l1b/deploy.shell [hunk 3]
```diff
@@ -0,0 +1,101 @@
+stream deploy --name ${S2USER}-l1a-inventory-ds --platformName ${PLATFORM_DEFAULT} --properties "\
```
Comment (glmxndr) [inline scripts/scdf-shell/inventory-l1b/deploy.shell:1]: Stream name seems incoherent.

File: scripts/scdf-shell/inventory-l1b/deploy.shell [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    deployer.*.kubernetes.readinessProbePath=/actuator/health/readiness,\
+    deployer.*.kubernetes.pod-security-context.fs-group=0,\
+    deployer.*.kubernetes.pod-security-context.run-as-user=0,\
+    deployer.*.kubernetes.limits.cpu=16,\
+    deployer.*.kubernetes.limits.memory=96Gi,\
+    deployer.*.kubernetes.requests.cpu=8100m,\
+    deployer.*.kubernetes.requests.memory=64Gi\
+"
+
+stream deploy --name ${S2USER}-l1a-inventory-gr --platformName ${PLATFORM_DEFAULT} --properties "\
```
Comment (glmxndr) [inline scripts/scdf-shell/inventory-l1b/deploy.shell:52]: Stream name seems incoherent.

**Review by glmxndr — CHANGES_REQUESTED** [review]
Stream name seems incoherent.

**Comment by BenEsquis** [comment]
Corrected the typos

**Review by glmxndr — APPROVED** [review]

**Pull request merged**